File: public/save.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $customString = $_POST["customString"];
    $idandrow = explode("|",$customString);
    $id = $idandrow[0];
    $row = explode(";",$idandrow[1]);
    
    $filename = '/media/stimtool_online/ABCFloat/'.$id.'_somatomapdata.csv';
    
    // open csv file for writing
    $f = fopen($filename, 'a');
    
    if ($f === false) {
    	die('Error opening the file ' . $filename);
        // die(phpinfo());
    }
    
    // write each row at a time to a file
    fputcsv($f, $row);
    // close the file
    fclose($f);
    
    // Redirect back to the HTML page
    header("Location: index.html");
    exit;
}
?>

File: public/save.py
from flask import Flask, request, redirect

app = Flask(__name__)

@app.route('/', methods=['POST'])
def handle_post():
    if request.method == 'POST':
        custom_string = request.form['customString']
        id_and_row = custom_string.split("|")
        id = id_and_row[0]
        row = id_and_row[1].split(";")
        
        filename = id + '_somatomapdata.csv'
        
        # open csv file for writing
        with open(filename, 'a') as f:
            # write each row at a time to a file
            f.write(','.join(row) + '\n')
        
        # Redirect back to the HTML page
        return redirect("index.html")

if __name__ == '__main__':
    app.run(debug=True)
